pkg/reconciler/kubernetes/tektonhub/tektonhub.go

```diff
@@ -496,6 +533,24 @@ func (r *Reconciler) transform(ctx context.Context, manifest mf.Manifest, th *v1
 	return &manifest, nil
 }
 
+func (r *Reconciler) checkPVC(ctx context.Context, th *v1alpha1.TektonHub, name string) (*corev1.PersistentVolumeClaim, error) {
+	pvc, err := r.kubeClientSet.CoreV1().PersistentVolumeClaims(th.Spec.GetTargetNamespace()).Get(ctx, name, metav1.GetOptions{})
+	if err != nil {
+		return nil, err
+	}
+
+	return pvc, nil
+}
+
+func (r *Reconciler) checkPVCOwnerRef(pvc *corev1.PersistentVolumeClaim, th *v1alpha1.TektonHub) bool {
+	if len(pvc.GetOwnerReferences()) == 1 {
+		if pvc.GetOwnerReferences()[0].Kind == th.Kind {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
add a to be deprecated comment arround this function and each place it is called, as we will not need it in future releases.

something like

```golang
// TODO: remove this after operator <x.y.z>
```